Add a proper Blog (#1902)
near/docs: blog/2024-04-21.md

---
title: We have a blog now!
authors: [gagdiez]
slug: we-have-a-blog
tags: [updates]
---

*Check check check. Is this thing on? Hello, world!*

<p><img src="/assets/images/protocol-b73c2a3ace3307226ee7eb2149ee432f.png" /></p>

<!-- truncate -->

## Hello there!

> Welcome to the new NEAR documentation blog! We're excited to inaugurate this new space were we can share news and updates. We know what you are thinking... why a blog? well, because it allows us to interact with you in a different way!

Reviewer: ```suggestion
Welcome to the new NEAR documentation blog! We're excited to inaugurate this new space where we can share news and updates. We know what you are thinking — why a blog? Well, it allows us to interact with you in a different way!
```

> You'll see, when writing documentation it is important for us to keep the content focused and concise. This way, when you are reading it, you have nothing to distract you, and you can focus on learning a new concept.

Reviewer: ```suggestion
When writing documentation, it is important for us to keep the content focused and concise. This way, when you read it, you have nothing to distract you, and you can focus on learning a new concept.
```

> The problem with this is that many times we wanted to share ideas, thoughts, or insights into why some things are the way they are. However, this would imply going over a tangent, explaining why some decisions where made, which will probably add noise to the document. Most of the times, users just want a link to an example, or working code... also, people don't like to read long texts.

Reviewer: ```suggestion
The problem with this is that many times, we want to share ideas, thoughts, or insights into why some things are the way they are. However, this would imply going off on a tangent and explaining why some decisions were made, which will probably add noise to the document. Most of the time, users just want a link to an example or working code. Also, people don't like to read long texts.
```

> Besides that, sometimes we just want to share what we did during the week. For example, did you notice that we fixed the nightmare that was the URL structure? Or that we added new landing pages for all concepts? We want to share these things with you, but they don't really belong in the docs.

Reviewer: ```suggestion
Besides that, sometimes we just want to share what we did during the week. For example, did you notice that we fixed the nightmare that was the URL structure, or that we added new landing pages for all concepts? We want to share these things with you, but they don't really belong in the Docs.
```

:::tip

> You would not believe the amount of times we said during a review "lets remove that, we are writing technical documentation, not a blog post"... well, now we can make the blog post!

Reviewer: ```suggestion
You would not believe the number of times we said during a review _"let's remove that, we are writing technical documentation, not a blog post"_ ... well, now we can make the blog post!
```

:::

## What to expect

> We will be using this space to share updates about our docs, as well as **complimenting them** with additional information that we think you'll find interesting.

Reviewer: ```suggestion
We will be using this space to share updates about our docs, as well as **complement them** with additional information that we think you'll find interesting.
```

Our aim is to keep the blog updated at least once a week, but given the limited amount of time we have, we will see how that goes.

> Moreover, we want to remind you that our docs are an open source and **collaborative project**. If you feel like you have something to share, or want to contribute to the blog, feel free to reach out. And by reach out we mean: [opening a PR](https://github.com/near/docs/pulls)!

Reviewer: ```suggestion
Moreover, we want to remind you that our Docs are an open source and **collaborative project**. If you feel like you have something to share, or want to contribute to the blog, feel free to reach out. And by reaching out, we mean [opening a PR](https://github.com/near/docs/pulls)!
```
